openrtb2/banner_ad_type.go

```diff
@@ -1,13 +0,0 @@
-package openrtb2
-
-// 5.2 Banner Ad Types
-//
-// Types of ads that can be accepted by the exchange unless restricted by publisher site settings.
-type BannerAdType int8
-
-const (
-	BannerAdTypeXHTMLTextAd   BannerAdType = 1 // XHTML Text Ad (usually mobile)
-	BannerAdTypeXHTMLBannerAd BannerAdType = 2 // XHTML Banner Ad. (usually mobile)
-	BannerAdTypeJavaScriptAd  BannerAdType = 3 // JavaScript Ad; must be valid XHTML (i.e., Script Tags Included)
-	BannerAdTypeIframe        BannerAdType = 4 // iframe
-)
```

Comment:
TODO: maybe import adcom1 here or bring back the enum. It's still nice to have human-readable constants in code.

Reply:
No Adcom1 counterpart, just bring it back.